### ISSUE TITLE
/dev/kvm is not group 'kvm'

### ISSUE THREAD

**Issue by mischief** [coreos/bugs#1130]
on coreos, the core user is in the group kvm but /dev/kvm is not group kvm.

```
core@lan-lab-3 / $ id
uid=500(core) gid=500(core) groups=500(core),10(wheel),78(kvm),150(sudo),233(docker),248(systemd-journal),250(portage),251
core@lan-lab-3 / $ ls -l /dev/kvm
crw------- 1 root root 10, 232 Feb 19 23:07 /dev/kvm
```

this means running qemu w/ kvm requires root.

**Comment by mischief** [comment]
probably just needs a tmpfiles snippet from somewhere.

**Comment by marineam** [comment]
No, a udev rule. Normally shipped in the qemu package.

**Comment by bgilbert** [comment]
Fixed in coreos/init@76a5454244071c954e0757bc639f07b3cd42dcfb.
